bftengine/src/bftengine/TimeUtils.hpp

```diff
@@ -0,0 +1,40 @@
+// Concord
+//
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
```

Comment:
typo: 2019

Reply:
Good catch. I changed it to 2018-2019, since this was based on an older file.